zion/src/main/scala/edu/uci/ics/cloudberry/zion/actor/BerryClient.scala

```diff
@@ -54,9 +54,12 @@ class BerryClient(val jsonParser: JSONParser,
     case initial: Initial if initial.ts == curKey =>
       val queryInfos = initial.queries.zip(initial.infos).map {
         case (query, info) =>
-          val bound = query.getTimeInterval(info.schema.timeField).getOrElse(new TInterval(info.dataInterval.getStart, DateTime.now))
+          val bound = info.schema.timeField match {
```

Comment:
I think we assume `timeField` is a necessary field, under what circumstance it could be `None`?